web/themes/custom/accessibility_theme/src/patterns/components/heading/heading.twig

```diff
@@ -0,0 +1,12 @@
+{{ attach_library('accessibility_theme/heading') }}
+
+{# Defaults to h2 (heading level 2) if no heading level is supplied. #}
+<h{{ heading_level|default('2') }} class="heading {{ modifier }}" {{ extra_attributes }}>
```

Comment:
What is **extra_attributes**?  We can remove as we typically address drupal attributes differently.  Use the method used in our training: https://mariohernandez.gitbook.io/components/chapter-3/drupal-attributes